src/platform-implementation-js/views/conversations/attachment-card-view.ts

```diff
@@ -44,21 +44,7 @@ export default class AttachmentCardView extends (EventEmitter as new () => Typed
     return this.#attachmentCardImplementation.getTitle();
   }
 
-  /**
-   * @deprecated  Please use the same-named method on the AttachmentCardClickEvent object instead.
-   */
   getDownloadURL(): Promise<string | null | undefined> {
```

Comment:
```suggestion
  /**
   * Get the URL for the attachment card's download link as a promise for a string.
   * For FILE attachment cards, the URL will be a short-lived URL that can be 
   * accessed without cookies. For CUSTOM attachment cards, the URL will be the    
   * downloadUrl property of the card's download button if it has one, otherwise 
   * null. Other attachment card types may not have a download URL, and the promise 
   * may resolve to null.
   */
```